build.sbt

```diff
@@ -329,6 +329,8 @@ lazy val scaldingCore = module("core").settings(
     "com.twitter" %% "bijection-macros" % bijectionVersion,
     "com.twitter" %% "chill" % chillVersion,
     "com.twitter" %% "chill-algebird" % chillVersion,
+    "com.twitter.elephantbird" % "elephant-bird-cascading2" % elephantbirdVersion,
```

Comment:
so, strictly speaking, I thought elephant-bird had GPL'd dependencies (lzo). That was actually one of the original reasons for not putting this in the core.

Do we need both of these dependencies here? 

I start to wonder if there is a single consumer of elephant-bird-cascading2 other than scalding. Looks like the answer is no:
https://mvnrepository.com/artifact/com.twitter.elephantbird/elephant-bird-cascading2/usages

Maybe we should move that code into this repo so we don't have the cross repo publish issue?

We could move it into a separate module, like we did with `maple` and just have the java code there. As far as elephant-bird-core, that is used elsewhere. I wonder if the gpl lzo code can be is a separate module in elephant-bird, then we could have `scalding-lzo` that depends on that and split `scalding-commons` into `core` and `lzo`?

Each new dependency is a possible pain point for others... So, I think we should add dependencies very carefully (and usually strive to remove them, not add them).